InfinityWorks.TechTest/Services/IFSAClient.cs

```diff
@@ -6,5 +6,7 @@ namespace InfinityWorks.TechTest.Services
     public interface IFsaClient
     {
         Task<FsaAuthorityList> GetAuthorities();
+
+        Task<AuthorityRatingItem> GetAuthorityRatingItems(int localAuthorityId);
```

Comment:
Shouldn't this be a list of ratings?

Reply:
The stub in the controller uses the AuthorityRatingItem object which it passes a list of, in json format, to the view. The IFsaClient is injected into the controller so this would be the list of objects the controller would expect when calling the GetAuthorityRatingItems method.

Reply:
Ignore my previous comment, realise what you mean now.